src/utils/rewards.ts

```diff
@@ -0,0 +1,7 @@
+import Big from "big.js";
+import { MS_PER_YEAR } from "./constants";
+
+export function calculateAnnualizedRewardAmount(amountPerBlock: Big, blockTimeMs: number): Big {
```

Comment:
Not sure where this is used, but would it make sense to get an rpc for that at some stage?

We've pushed these types of calcs for other scenarios to rpcs - granted some of them were more complex, so it made sense to keep the logic in the parachain code. Just something to consider

Reply:
> Yes, I think adding rpc that would provide calculation with more accurate average block time makes sense!

cc @gregdhill